index.html

```diff
@@ -6709,17 +6736,37 @@ <h2 id="AudioListener">
             <p>
               Sets the z coordinate position of the audio listener in a 3D
               Cartesian coordinate space. The default value is 0. This
-              parameter is a-rate.
+              parameter is <a>a-rate</a> when used with a <a>PannerNode</a>
+              that has a <a href=
+              "#widl-PannerNode-panningModel">panningModel</a> set to
+              <code>"equalpower"</code>, <a>k-rate</a> otherwise.
             </p>
           </dd>
           <dt>
             readonly attribute AudioParam forwardX
           </dt>
           <dd>
             <p>
+              The <code>forwardX, forwardY, forwardZ</code> parameters
+              represent a direction vector in 3D space. Both a
+              <code>forward</code> vector and an <code>up</code> vector are
+              used to determine the orientation of the listener. In simple
+              human terms, the <code>forward</code> vector represents which
+              direction the person's nose is pointing. The <code>up</code>
+              vector represents the direction the top of a person's head is
+              pointing. These values are expected to be linearly independent
+              (at right angles to each other), and unpredictable behavior may
+              result if they are not. For normative requirements of how these
+              values are to be interpreted, see the <a href=
+              "#Spatialization">spatialization section</a>.
+            </p>
```

Comment:
This paragraph is already at https://webaudio.github.io/web-audio-api/#AudioListener, 3rd paragraph, so this should be removed.